stub/src/test/java/io/grpc/stub/ClientCallsTest.java

```diff
@@ -329,17 +345,16 @@ public void onCompleted() {
     integerStreamObserver.request(3);
     integerStreamObserver.onCompleted();
     assertTrue(latch.await(5, TimeUnit.SECONDS));
-    // Very that number of messages produced in each onReady handler call matches the number
+    // Verify that number of messages produced in each onReady handler call matches the number
     // requested by the client. Note that ClientCalls.asyncBidiStreamingCall will request(1)
     assertEquals(Arrays.asList(0, 1, 1, 2, 2, 2), receivedMessages);
   }
 
-  @org.junit.Ignore
   @Test
   public void inprocessTransportOutboundFlowControl() throws Exception {
     final CountDownLatch latch = new CountDownLatch(1);
-    final Semaphore semaphore = new Semaphore(1);
```

Comment:
And it looks like the usage in `inprocessTransportInboundFlowControl()` was completely useless since no path could call acquire when the count was below zero. I'll glance at that more and remove it if it is indeed unnecessary.